cli/lsp/language_server.rs

```diff
@@ -1769,8 +1769,8 @@ impl Inner {
     let req = tsc::RequestMethod::FindRenameLocations((
       specifier,
       line_index.offset_tsc(params.text_document_position.position)?,
-      true,
-      true,
+      false,
+      false,
       false,
     ));
```

Comment:
By looking at this code alone, it's hard to tell what this means. It would be nice if we didn't use tuples here.

These are the arguments for `find_in_comments` and `find_in_strings`.

Reply:
I'd be fine using q struct with the more complicated APIs. It gets unwieldy for all of them though.

Reply:
Done in 8940d22.